canonical-web-and-design/charmhub.io
# 55 store homepage update filter navigation

## Done

- Replace side nav with checkbox filter list on store page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e849151aec09346e01c3c8c)


## Issue / Card

Fixes
